### PR TITLE
Pioneer 15 Updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Deploy csv updates per Pioneer 15 QLR (3204-01502 in Vault):
CP05MOAS-PG583_Deploy.csv - glider recovery
CP01CNPM_Deploy.csv - CNPM-4 deployment
CP03ISPM_Deploy.csv - ISPM-4 deployment
CP03ISSM_Deploy.csv - ISSM-11 recovery and ISSM-12 deployment; also 2 SN updates for ISSM-12 - NSIF FLORT (from 1216 to 1214) and NSIF PHSEN (from P0100 to P0095).  Verify via ISSM-12 config in Vault.
CP01CNSM_Deploy.csv - CNSM-13 deployment only so far

### CODE REVIEW

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 1]
```diff
@@ -13,10 +13,10 @@ TS20200613,,MZ20200731,,CP05MOAS-PG583-03-FLORTM000,2,1,2020-06-13T20:10:46,2020
...
+TS20200819,,AR48,,CP05MOAS-PG583-01-CTDGVM000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-CTDGVM-09307,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,AR48,,CP05MOAS-PG583-02-DOSTAM000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-DOSTAM-00539,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,AR48,,CP05MOAS-PG583-03-FLORTM000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTM-04093,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,AR48,,CP05MOAS-PG583-04-FLORTO000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTO-01348,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,AR48,,CP05MOAS-PG583-05-NUTNRM000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-NUTNRM-00709,40.03803,-70.69827,,200,200,CNPM
+TS20200819,,AR48,,CP05MOAS-PG583-06-PARADM000,3,1,2020-08-19T19:25:00,2020-10-29T15:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-PARADM-50218,40.03803,-70.69827,,200,200,CNPM
```
Comment (snwhite) [inline deployment/CP05MOAS-PG583_Deploy.csv:22]: Looks good.

**Review by snwhite — APPROVED** [review]

File: deployment/CP03ISSM_Deploy.csv [hunk 2]
```diff
@@ -310,65 +310,65 @@ AR34,,AR39,,CP03ISSM-SBD11-06-METBKA000,10,1,2019-04-07T14:42:00,2019-10-01T11:5
...
+AR48,,,,CP03ISSM-RID27-04-DOSTAD000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-DOSTAD-00387,40.367077,-70.881735,,7,91,
+AR48,,,,CP03ISSM-SBC11-00-CPMENG000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGCON-BCPM01-50004,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD11-00-DCLENG000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGCON-BDCL11-50004,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD11-01-MOPAK0000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-MOPAK0-11839,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD11-02-HYDGN0000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-41692,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD11-06-METBKA000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-METLGR-00033,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD12-00-DCLENG000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGCON-BDCL12-50004,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD12-03-HYDGN0000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-49868,40.367077,-70.881735,,0,91,
+AR48,,,,CP03ISSM-SBD12-04-PCO2AA000,12,1,2020-10-31T19:31:00,,CGMCP-03ISSM-00012,,CGINS-PCO2AA-33182,40.367077,-70.881735,,0,91,
```
[the remark this answers is not in the document]
Comment (reedan88) [reply deployment/CP03ISSM_Deploy.csv:374]: Looks good.

File: deployment/CP05MOAS-PG583_Deploy.csv (hunk 1 above)
Comment (reedan88) [reply deployment/CP05MOAS-PG583_Deploy.csv:22]: Looks good.

**Review by reedan88 — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 15 AM Updates [DT-102]](https://trello.com/c/PK5rFp9p/102-pioneer-15-am-updates-dt-102)

**Pull request merged**